docs/_guides/codewind-eclipse-quick-guide.md

```diff
@@ -9,17 +9,14 @@ duration: 5 minutes
 tags: Codewind, Eclipse, microservice
 ---
 
-# Getting Started with Codewind in Eclipse
-
 ## Objectives
 * Install Eclipse and Codewind
-* Develop a simple microservice, using Eclipse Codewind in Eclipse
-
+* Develop a simple microservice that uses Eclipse Codewind in Eclipse
 
 ## Overview
-Eclipse Codewind provides the ability to create application projects from `Application Stacks` that your company has built, enabling developers to focus on their code and not infrastructure and Kubernetes.  Application deployments to Kubernetes occur via pipelines when developers commit their local code to the correct Git repos Kabanero is managing via webhooks.
+Use Eclipse Codewind to create application projects from `Application Stacks` that your company builds. With Codewind, you can focus on your code and not on infrastructure and Kubernetes. Application deployments to Kubernetes occur through pipelines when developers commit their local code to the correct Git repos Kabanero is managing via webhooks.
```

Comment:
- I don't think Application Stacks needs to be in code phrase format. It's not text that a user is inputting, nor is it code itself.
- Latin phrases like "via" are best replaced with an English equivalent, such as "with." I would have hoped Acrolinx would catch this, but it's something we can watch out for.